docs/installation/configuration/opennotifs_config.rst

```diff
@@ -55,21 +53,72 @@ using Open Zaak as the example.
 
    d. Click **Opslaan**.
 
-2. Finally, we need to allow Open Zaak to access Open Notificaties (for
+2. Then we need to allow Open Zaak to access Open Notificaties (for
    authentication purposes, so we can then check its authorizations):
 
    a. Navigate to **API Autorisaties > Client credentials**
    b. Click **Client credential toevoegen**.
    c. Fill out the form:
 
-      - **Client ID**: *The same Client ID as given in Open Zaak step 2c*
-      - **Secret**: *The same Secret as given in Open Zaak step 2c*
+      - **Client ID**: *For example:* ``open-zaak``
+      - **Secret**: *Some random string*
 
       Make sure Open Zaak is configured to use this Client ID and secret to
       access Open Notificaties.
 
    d. Click **Opslaan**.
 
+3. After that, we need to configure the **Notificatiescomponentconfiguratie**.
+
+   a. Navigate to **Configuratie > Notificatiescomponentconfiguratie** and add a new **Notifications api service**
+   b. Fill out the form and use the client ID and secret from step 1c
+   c. Click **Opslaan** on the **Service** form
+   d. Click **Opslaan** on the **Notificatiescomponentconfiguratie** form
+
+4. Finally, we need to add the client ID and secret from step 1c under **API Autorisaties > Autorisatiegegevens > Autorisatiegegeven toevoegen**
+
+Configure Open Zaak
+===================
+
+In case Open Zaak is not configured yet, refer to `configuration of Open Zaak`_. Use the following as input for the management command:
+
+   - **notifications-api-app-client-id**: client ID from step 1c above
+   - **notifications-api-app-secret**: secret from step 1c above
+   - **notifications-api-client-id**: client ID from step 2c above
+   - **notifications-api-secret**: secret from from step 2c above
```

Comment:
The management command `setup_configuration` has been changed and now doesn't have these parameters. Could you either change them to env vars or describe how to use them for steps here https://open-zaak.readthedocs.io/en/stable/installation/config/openzaak_config.html (in case it's manual configuration)

Reply:
I changed it and mentioned the envvars instead